frontend/package-lock.json

```diff
@@ -0,0 +1,8843 @@
+{
```

Comment:
This file must be untracked as well so that it isn't pushed into the main branch